### DISCUSSION TITLE
今天突然被windows自带的杀毒软件报木马了

### DISCUSSION THREAD

**Discussion by kuroneko52999** [MaaAssistantArknights/MaaAssistantArknights#7660] [General]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/29587679/2ba3c750-8622-4b4d-8fb2-73c479f189fe)

**Comment by shiguang-wei** [comment]
而且使用过程中exe会消失，在D盘的原文件夹中找不到，搜索文件也没有这个文件，也不在回收站中，基本两天就要重装一次

**Comment by status102** [comment]
#6662
